cumulusci.yml

```diff
@@ -1207,6 +1174,113 @@ flows:
                     path: scripts/New_Testing_User.cls
                     apex: createTestingUser();
 
+    ldv_test_data:
+        description: Creates 1000 Household Contact records with associated Recurring Donations (RD2 format) and between 0 and 5 standard Donation opps on each, along with GAU's, Affliiation, Relationships, etc.. Use "-o generate_and_load_from_yaml__num_records {num}" to change the number of created Contacts & related records
+        group: NPSP
+        steps:
+            5:
+                task: execute_anon
+                ## Configure RD2 to not create installments, disable household addresses and enable automatic naming
+                options:
+                    path: datasets/rd2/config_npsp_for_ldv_data_load.cls
+                    apex: before_data_load();
+            10:
+                task: disable_triggers
+                ## Disable Rollups for both the delete and the data load to avoid lock errors
+                ## Disable the RD2 Opp trigger to prevent it from attempting to create a new RD opp
+                options:
+                    handlers:
+                        - CRLP_Rollup_TDTM
+                        - RLLP_OppRollup_TDTM
+                        - RD2_RecurringDonationsOpp_TDTM
+                    active: false
+                    restore: false
+            15:
+                task: unschedule_apex
+                ## Delete all scheduled NPSP jobs ... just in case.
+            16:
+                task: set_duplicate_rule_status
```

Comment:
I love this sequence - it's an awesome example of using OOTB tools to achieve a complex objective and I'm definitely going to steal it for a presentation sometime.